src/frontend/main/MainWrapper.tsx

```diff
@@ -65,14 +65,18 @@ const Container = styled.div`
   margin: 0 auto;
 `;
 
-const TotalResponses = styled.div`
+const TotalResponses = styled(Container)<{ topBorder?: boolean }>`
+  border-top: ${({ topBorder, theme }) => (topBorder ? `1px solid ${theme.black}` : 'none')};
+  padding: 10px 0;
+`;
+
+const TotalResponsesWrapper = styled.div`
   background: ${props => props.theme.white};
   position: fixed;
   bottom: 0px;
-  padding: 10px 0 10px 16px;
   font-size: 14px;
   font-style: italic;
-  width: 100vw;
```

Comment:
When this is set to 100vw, somehow I always get centered total responses div to be off by a few pixels compared to the tables, even though they share the same width and centerred using margin

![image](https://user-images.githubusercontent.com/6004147/81658576-93b32e00-9441-11ea-9b1f-9256e55fc928.png)